packages/angular_devkit/core/src/experimental.ts

```diff
@@ -5,6 +5,6 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
-import * as workspace from './workspace';
+import * as workspace from './workspace/index';
```

Comment:
The api guardian only used classic module resolution not to resolve external modules. Currently the index is resolve because we use node module resolution

Reply:
Then yes, I think we need a linting rule to prevent regressions here. I did a cursory search and couldn't find anything off-the-shelf. @hansl WDYT?